containers/proxy-helm/templates/config.yaml

```diff
@@ -9,8 +9,15 @@ data:
     server_version: "{{ .Values.server_version }}"
     proxy_fqdn: {{ .Values.proxy_fqdn }}
     email: {{ .Values.email }}
-    max_cache_size_mb: {{ .Values.max_cache_size_mb }}
```

Comment:
This still needs to be understood by the container config as there could be containers with this config in the wild.
The other solution is to not group the squid options under `squid_config`. Not sure which is best.

Reply:
Makes sense, I've removed the nesting of values